nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.github/workflows/ci.yml

```diff
@@ -1,27 +1,28 @@
 name: nf-core CI
 # This workflow is triggered on pushes and PRs to the repository.
 # It runs the pipeline with the minimal test dataset to check that it completes without any syntax errors
-on: [push, pull_request]  
+on: [push, pull_request]
 
 jobs:
   test:
-    env: 
+    env:
       NXF_VER: {% raw %}${{ matrix.nxf_ver }}{% endraw %}
       NXF_ANSI_LOG: false
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     strategy:
       matrix:
         # Nextflow versions: check pipeline minimum and current latest
         nxf_ver: ['19.10.0', '']
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
       - name: Install Nextflow
         run: |
           wget -qO- get.nextflow.io | bash
           sudo mv nextflow /usr/local/bin/
-      - name: Pull image
-        run: |
-          docker pull {{ cookiecutter.name_docker }}:dev && docker tag {{ cookiecutter.name_docker }}:dev {{ cookiecutter.name_docker }}:dev
-      - name: Run test
+      - name: Pull docker image
+        run: docker pull {{ cookiecutter.name_docker }}:dev && docker tag {{ cookiecutter.name_docker }}:dev {{ cookiecutter.name_docker }}:dev
```

Comment:
```suggestion
        run: |
```

Reply:
I think this is why the tests are failing. The `docker` command needs to be on its own line.